src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.CurlResponseMessage.cs

```diff
@@ -426,10 +431,25 @@ private void CheckDisposed()
                 }
             }
 
-            [Conditional(VerboseDebuggingConditional)]
-            private void VerboseTrace(string text = null, [CallerMemberName] string memberName = null)
+            private void EventSourceTrace<T>(string formatMessage, T arg0, [CallerMemberName] string memberName = null)
```

Comment:
Maybe stick in a quick comment to ensure someone like me doesn't come along and remove the generics.  :)